<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayWindow</class>
 <widget class="QMainWindow" name="DisplayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_close">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tab_sci_com_conn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_sci_main">
       <attribute name="title">
        <string>Scientists</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="6">
         <widget class="QLineEdit" name="line_search_sci">
          <property name="placeholderText">
           <string>Search...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="6" rowspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;&lt;br/&gt;&lt;br/&gt;S&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;C&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;I&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;E&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;N&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;T&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;I&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;S&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;T&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;S&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="6">
         <widget class="QTableWidget" name="table_display_sci">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Year of Birth</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Year of Death</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Gender</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Year of Birth</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Year of Death</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Gender</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QLineEdit" name="line_sci_name_add_remove_edit">
          <property name="placeholderText">
           <string>Enter Full Name...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="line_sci_birth_add_remove_edit">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Birth...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QLineEdit" name="line_sci_death_add_remove_edit">
          <property name="placeholderText">
           <string>Death?...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QLineEdit" name="line_sci_gender_add_remove_edit">
          <property name="placeholderText">
           <string>Sex?...</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_add_sci">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="pushButton_remove_sci">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2" colspan="2">
         <widget class="QPushButton" name="pushButton_edit_sci">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_com">
       <attribute name="title">
        <string>Computers</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QLineEdit" name="line_com_built_add_remove_edit">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Yes/No</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton_com_add">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLineEdit" name="line_com_name_add_remove_edit">
          <property name="placeholderText">
           <string>Enter Computer Name...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Built?</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="pushButton_com_remove">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pushButton_com_edit">
          <property name="text">
           <string>Edit</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="5">
         <widget class="QLineEdit" name="line_search_com">
          <property name="placeholderText">
           <string>Search...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="5">
         <widget class="QTableWidget" name="table_display_com">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Build Year</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Built</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="line_com_type_add_remove_edit">
          <property name="placeholderText">
           <string>Enter Type...</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="line_com_year_add_remove_edit">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Enter Year...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Build Year</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;C&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;O&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;M&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;P&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;U&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;T&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;E&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;R&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;S&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_conn">
       <attribute name="title">
        <string>Connections</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QTabWidget" name="tab_connect_display_edit">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab_conn_display">
           <attribute name="title">
            <string>Display</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLineEdit" name="line_connect_search">
              <property name="placeholderText">
               <string>Search...</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QTableWidget" name="table_display_connect">
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Scientist Name</string>
               </property>
               <property name="textAlignment">
                <set>AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Computer Name</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_conn_edit">
           <attribute name="title">
            <string>Edit</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="2" column="3">
             <widget class="QLineEdit" name="line_connect_sci_gender_edit">
              <property name="placeholderText">
               <string>Sex?...</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="4">
             <widget class="QTableWidget" name="table_edit_connect_com">
              <column>
               <property name="text">
                <string>Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Build Year</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Type</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Built</string>
               </property>
              </column>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Computer Name</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="line_connect_sci_death_edit">
              <property name="placeholderText">
               <string>Death?...</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLineEdit" name="line_connect_sci_name_edit">
              <property name="placeholderText">
               <string>Enter Full Name...</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="line_connect_sci_birth_edit">
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Birth...</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Build Year</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="line_connect_com_year_edit">
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Enter Year...</string>
              </property>
             </widget>
            </item>
            <item row="5" column="2">
             <widget class="QLineEdit" name="line_connect_com_type_edit">
              <property name="placeholderText">
               <string>Enter Type...</string>
              </property>
             </widget>
            </item>
            <item row="4" column="3">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Built?</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>Type</string>
              </property>
             </widget>
            </item>
            <item row="5" column="3">
             <widget class="QLineEdit" name="line_connect_com_built_edit">
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Yes/No</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLineEdit" name="line_connect_com_name_edit">
              <property name="placeholderText">
               <string>Enter Computer Name...</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="pushButton_connect">
              <property name="text">
               <string>Connect</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Year of Death</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Gender</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Scientists Name</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Year of Birth</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="4">
             <widget class="QTableWidget" name="table_edit_connect_sci">
              <column>
               <property name="text">
                <string>Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Year of Birth</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Year of Death</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Gender</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;C&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;O&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;N&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;N&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;E&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;C&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;T&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;I&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;O&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;N&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;S&lt;/span&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>line_search_com</tabstop>
  <tabstop>line_com_name_add_remove_edit</tabstop>
  <tabstop>line_com_year_add_remove_edit</tabstop>
  <tabstop>line_com_type_add_remove_edit</tabstop>
  <tabstop>line_com_built_add_remove_edit</tabstop>
  <tabstop>pushButton_com_add</tabstop>
  <tabstop>pushButton_com_remove</tabstop>
  <tabstop>pushButton_com_edit</tabstop>
  <tabstop>pushButton_remove_sci</tabstop>
  <tabstop>pushButton_edit_sci</tabstop>
  <tabstop>tab_sci_com_conn</tabstop>
  <tabstop>table_display_com</tabstop>
  <tabstop>line_search_sci</tabstop>
  <tabstop>table_display_sci</tabstop>
  <tabstop>line_sci_name_add_remove_edit</tabstop>
  <tabstop>line_sci_birth_add_remove_edit</tabstop>
  <tabstop>line_sci_death_add_remove_edit</tabstop>
  <tabstop>line_sci_gender_add_remove_edit</tabstop>
  <tabstop>pushButton_add_sci</tabstop>
  <tabstop>tab_connect_display_edit</tabstop>
  <tabstop>line_connect_search</tabstop>
  <tabstop>table_display_connect</tabstop>
  <tabstop>line_connect_sci_gender_edit</tabstop>
  <tabstop>table_edit_connect_com</tabstop>
  <tabstop>line_connect_sci_death_edit</tabstop>
  <tabstop>line_connect_sci_name_edit</tabstop>
  <tabstop>line_connect_sci_birth_edit</tabstop>
  <tabstop>line_connect_com_year_edit</tabstop>
  <tabstop>line_connect_com_type_edit</tabstop>
  <tabstop>line_connect_com_built_edit</tabstop>
  <tabstop>line_connect_com_name_edit</tabstop>
  <tabstop>pushButton_connect</tabstop>
  <tabstop>table_edit_connect_sci</tabstop>
  <tabstop>pushButton_close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
